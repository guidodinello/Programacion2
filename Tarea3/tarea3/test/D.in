# Prueba D
insertarEnBinario (11,11.11)
insertarEnBinario (8,8.8)
insertarEnBinario (12,12.12)
esAvl
insertarEnBinario (13,-13.13)
insertarEnBinario (14,14.14)
esAvl
alturaBinario
linealizacion
sumaUltimosPositivos 3
menores 12
nivelEnBinario 2
esCamino 3 (11,1.1) (12,1.1) (13,1.1)
imprimirBinario
mayor
removerMayor
removerDeBinario 11
raiz
Fin