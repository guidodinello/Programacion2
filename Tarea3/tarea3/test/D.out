1>#  Prueba D.
2>Insertado.
3>Insertado.
4>Insertado.
5>AVL.
6>Insertado.
7>Insertado.
8>No AVL.
9>4.
10>(8,8.80)(11,11.11)(12,12.12)(13,-13.13)(14,14.14)
11>37.37
12>
-(13,-13.13)
(11,11.11)
-(8,8.80)
13>(8,8.80)(12,12.12)
14>NO es camino.
15>
---(14,14.14)
--(13,-13.13)
-(12,12.12)
(11,11.11)
-(8,8.80)
16>(14,14.14)
17>Removido el mayor.
18>Removido.
19>(8,8.80)
20>Fin.
